installing/disconnected_install/installing-mirroring-disconnected.adoc

```diff
@@ -140,3 +140,9 @@ include::modules/oc-mirror-image-set-config-examples.adoc[leveloffset=+1]
 
 // Command reference for oc-mirror
 include::modules/oc-mirror-command-reference.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+[id="additional-resources_installing-mirroring-disconnected"]
+== Additional resources
+
+* See xref:../../updating/updating-restricted-network-cluster/index.adoc#about-restricted-network-updates[Updating a disconnected environment] for information about performing an update in a disconnected environment.
```

Comment:
I'd recommend clarifying that we're talking about an OCP update (not an update to the mirror registry). You should also use the actual title of the section you're linking to, which looks like "About disconnected environment updates". Maybe:

```suggestion
* See xref:../../updating/updating-restricted-network-cluster/index.adoc#about-restricted-network-updates[About disconnected environment updates] for information about updating a cluster in a disconnected environment.
```

Or maybe this isn't going to the page you intended?

Reply:
To that point, I was intending to point users, in a sense, towards the entirety of the "Updating a disconnected environment" section, even if you technically can only click on the index/about page rather than the section itself. If it's better practice to name the specific page I'm linking to instead, I'll do that.

Implementing both suggestions, thanks!